core/src/test/java/nl/nn/adapterframework/configuration/classloaders/ClassLoaderTestBase.java

```diff
@@ -254,4 +254,12 @@ public void configurationFileCustomLocationAndBasePath() throws Exception {
 		assertNotNull("config file ["+configFile+"] cannot be found", configURL);
 		assertTrue(configURL.getPath().endsWith(file));
 	}
+
+	@Test
+	public void toStringTest() throws Exception {
+		String logPrefix = classLoader.getClass().getSimpleName() + "@" + Integer.toHexString(classLoader.hashCode());
+
+		//Should match DatabaseClassLoader@1234abcd[<CONFIG-NAME>]
+		assertEquals(logPrefix+"["+getConfigurationName()+"]", classLoader.toString());
```

Comment:
Werkt dit altijd? Of alleen voor IConfigurationClassLoaders? Dan class hernoemen.